src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/EnvelopeRetrieverService.java

```diff
@@ -22,23 +23,27 @@
 
     private final EnvelopeRepository envelopeRepository;
     private final EnvelopeAccessProperties envelopeAccessProperties;
+    private final EnvelopeResponseMapper envelopeResponseMapper;
 
     public EnvelopeRetrieverService(
         EnvelopeRepository envelopeRepository,
         EnvelopeAccessProperties envelopeAccessProperties
     ) {
         this.envelopeRepository = envelopeRepository;
         this.envelopeAccessProperties = envelopeAccessProperties;
+        this.envelopeResponseMapper = new EnvelopeResponseMapper();
```

Comment:
We may as well make all the methods in `EnvelopeResponseMapper`.
What do others think?

Reply:
I am not a big fan of static methods in object oriented languages even if in some cases they can be really useful. In this case it seems to me it's debatable which way is better.

Reply:
if you prefer non static methods, perhaps we could at least inject an object of this instance instead of creating it inside?

Reply:
Yes, it could be injected but it's a plain old (boring) java object that holds no state. The reason why I did not make it a "component" is because I can avoid using the SpringRunner for my unit tests, but again that is really a matter of taste.